sandbox/registration-processor-mz.properties

```diff
@@ -128,6 +128,8 @@ RIDGENERATION=http://kernel-ridgenerator-service/v1/ridgenerator/generate/rid
 # We specify Ingress location for dmz which is any worker node and a ingress port
 SYNCSERVICE=${dmz.ingress.base.url}/registrationprocessor/v1/registrationstatus/sync
 PACKETRECEIVER=${dmz.ingress.base.url}/registrationprocessor/v1/packetreceiver/registrationpackets
+# this property is used in lostrid api to get postal code 
+mosip.registration.processor.postalcode.req.url=${dmz.ingress.base.url}/v1/masterdata/registrationcenters
 
```

Comment:
Also change the below propery in the dmz properties file

mosip.registration.processor.postalcode.req.url=${mz.ingress.url}/v1/masterdata/registrationcenters

Reply:
taken care

Reply:
I don't see the changes to dmz property file in this PR

Reply:
ok resolved